wizard/src/types.ts

```diff
@@ -91,12 +91,18 @@ export interface CsvData {
 
 // Types used for Detection.
 
-// DetectedFormat denotes the format type associated with some detected type..
-export interface DetectedFormat {
-  propertyName: string;
+// An abstraction for a Data Commons entity, e.g. a place type or property.
+interface Entity {
+  dcName: string;
   displayName: string;
 }
 
+// A Data Commons entity type, e.g. Country (which a type of Place).
+export type DCType = Entity;
```

Comment:
wondering if its necessary to define these new types DCType and DCProperty ..

Reply:
I don't think they are necessary. But I feel it's more readable and obvious what they are. Otherwise, it is easy to mistakenly swap the Property entity (DCProperty) with the Type entity (DCType).